### PR TITLE
Added styling to Resend Verification Email Page &  Password Recovery …

### PR DESCRIPTION
**Pull request by STR1ND3R**

### CODE REVIEW

File: app/views/users/confirmations/new.html.erb [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
+    <div class="mt-3">
+      <%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
+        <%= link_to "Forgot your password?", new_password_path(resource_name) %>
+      <% end %>
+    </div>
   </div>
-<% end %>
 
-<%= render "users/shared/links" %>
+</div>
```
Comment (MauricioNC) [inline app/views/users/confirmations/new.html.erb:37]: Add an empty line at the end of your code

File: app/views/users/passwords/new.html.erb [hunk 2]
```diff
@@ -1,16 +1,38 @@
...
+      <% end %>
+    </div>
 
-  <div class="actions">
-    <%= f.submit "Send me reset password instructions" %>
   </div>
-<% end %>
 
-<%= render "users/shared/links" %>
+</div>
```
Comment (MauricioNC) [inline app/views/users/passwords/new.html.erb:38]: Add an empty line at the end of your code

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/views/users/confirmations/new.html.erb (hunk 1 above)
Comment (STR1ND3R) [reply app/views/users/confirmations/new.html.erb]: done

File: app/views/users/passwords/new.html.erb (hunk 2 above)
Comment (STR1ND3R) [reply app/views/users/passwords/new.html.erb]: done

**Review by MauricioNC — APPROVED** [review]

**Review by JuanequeX — APPROVED** [review]

**Review by PudinDeveloper — APPROVED** [review]

**Pull request merged**